src/python/pants/backend/python/rules/run_python_binary.py

```diff
@@ -45,30 +47,49 @@ async def create_python_binary_run_request(
 
     transitive_targets = await Get(TransitiveTargets, Addresses([field_set.address]))
 
+    # Note that we get an intermediate PexRequest here (instead of going straight to a Pex)
+    # so that we can get the interpreter constraints for use in ipython_request.
```

Comment:
Bad copy pasta.

Reply:
While you're fixing this, can you please apply this diff:

```diff
diff --git a/src/python/pants/backend/python/rules/pex.py b/src/python/pants/backend/python/rules/pex.py
index 4258b6dc0..d4160928d 100644
--- a/src/python/pants/backend/python/rules/pex.py
+++ b/src/python/pants/backend/python/rules/pex.py
@@ -186,7 +186,6 @@ class PexPlatforms(DeduplicatedCollection[str]):
 
     @classmethod
     def create_from_platforms_field(cls, field: PythonPlatformsField) -> "PexPlatforms":
-        # TODO(#9562): wire to `--python-setup-platforms` once we know when/where it should be used.
         return cls(field.value or ())
 
     def generate_pex_arg_list(self) -> List[str]:

```

I missed it in my PR yesterday to remove `--python-setup-platforms`.